src/libvx.c

```diff
@@ -352,10 +357,12 @@ vx_error vx_open(vx_video** video, const char* filename, const vx_video_options
 	// Open stream
 	int open_result = avformat_open_input(&me->fmt_ctx, filename, NULL, NULL);
 	if (open_result != 0) {
-		if (open_result == AVERROR(ENOENT) || open_result == AVERROR(EINVAL)) {
```

Comment:
`AVERROR(EINVAL)` (invalid argument) occurred if the file path was empty, but also for a lot of other reasons e.g. if the file was corrupt. It is better to only return `VX_ERR_FILE_NOT_FOUND` on `AVERROR(ENOENT)` and to instead log and handle other errors with a more general VX_ERR